db/dataset_eval.py

```diff
@@ -11,12 +11,14 @@
 STATUS_RUNNING = "running"
 STATUS_DONE = "done"
 STATUS_FAILED = "failed"
-
+# Eval locations are defined in `eval_location_type` type. See schema definition.
```

Comment:
Think carefully about what you want to say in comments. They normally shouldn't just duplicate what the code says. One suggestion people make about comments is "say why, not what". You can read the code to see what it does.
Don't think that you have to copy a comment from similar code if you don't think that it gives any value.
I would consider writing here something like

`# Location to run an evaluation (on the AB server of the user's local server), defined in postgres type 'eval_location_type'`